crime-commons-classes/src/main/java/uk/gov/justice/laa/crime/filter/mdc/AddLaaTransactionIdToMDC.java

```diff
@@ -31,12 +24,13 @@ public void doFilter(ServletRequest request,
 
     String laaTransactionIdHeaderValue = extractLaaTransactionIdFromHeader(request);
 
-    if(StringUtils.isNotBlank(laaTransactionIdHeaderValue)){
+    if (StringUtils.isNotBlank(laaTransactionIdHeaderValue)) {
       log.debug("Adding the {} [{}] to the MDC", LAA_TRANSACTION_ID, laaTransactionIdHeaderValue);
       MDC.put(LAA_TRANSACTION_ID, laaTransactionIdHeaderValue);
 
     } else {
-      log.debug("Not adding the {} [{}] to the MDC, as it is blank", LAA_TRANSACTION_ID, laaTransactionIdHeaderValue);
+      log.debug("Not adding the {} [{}] to the MDC, as it is blank", LAA_TRANSACTION_ID,
```

Comment:
Do we need to pass laaTransactionIdHeaderValue as an argument as we know this is not available here?

Reply:
I've a personal preference for doing it like this, as if a developer changes the if condition above
`StringUtils.isNotBlank(laaTransactionIdHeaderValue)`
but doesn't update the hardcoded `laaTransactionIdHeaderValue` in the log statement it will be inconsistent.